src/app/frontend/chrome/chrome.scss

```diff
@@ -22,6 +22,7 @@
 
 .kd-toolbar {
   box-shadow: $whiteframe-shadow-z1;
+  min-height: 64px;
```

Comment:
You can leave default behavior on, and move the FAB a few pixels up on smaller devices. You can use media queries for this https://material.angularjs.org/latest/layout/introduction